app/models/scenario.rb

```diff
@@ -26,18 +26,16 @@ def destroy_with_mode(mode)
   end
 
   def self.icons
-    @icons ||= begin
-      YAML.load_file(Rails.root.join('config/icons.yml'))
-    end
+    @icons ||= YAML.load_file(Rails.root.join('config/icons.yml'))
```

Comment:
Don't we need `unsafe` here as well? We do have symbols in the [yml](https://github.com/huginn/huginn/blob/7b61253fa261486c3a9f89e4c71c3aaa34069be5/config/icons.yml)

Reply:
Psych.load has Symbol in its default list of `permitted_classes`, so it is OK.